install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

```diff
@@ -14,14 +14,13 @@ spec:
   template:
     metadata:
       labels:
-        app: {{ template "grafana.name" . }}
-        release: {{ .Release.Name }}
```

Comment:
I think there is value to keep release label so users can easily query what makes up a helm release.

Reply:
This breaks upgrading from istio.yaml ( release / upgradeVersion ) to istio.yaml (helm ), which is 
pretty important for now.

We can add annotations or other markers. I'm also not sure if the cost is worth it (k8s keeps indexes by label)